src/Components/Facility/Investigations/ShowInvestigation.tsx

```diff
@@ -80,6 +80,11 @@ export default function ShowInvestigation(props: any) {
     },
   );
 
+  const { data: consultation } = useQuery(routes.getConsultation, {
+    pathParams: { id: consultationId },
+    prefetch: !!consultationId,
+  });
```

Comment:
Can we pass the consultation object as prop to this component instead of making API call just for checking whether to disable or not?